.yarn/plugins/@yarnpkg/plugin-interactive-tools.cjs

```diff
@@ -0,0 +1,77 @@
+/* eslint-disable */
```

Comment:
What is this? Why is is necessary to bundle? Can't we just use NPM that everyone has installed anyway?

Reply:
Various parts of this project came from the polkadot/api or acala/laminar/moonbase sdk repository while was getting an idea on ​​how to extend the types and api in a project with yarn. I have removed most of these declarations that i see now they are not necessary. Same with prettier, eslint from @polkadot/dev.